tests/ai/dataset_pipeline/test_voice_quality_assessor.py

```
@@ -2,29 +2,32 @@
     from pydub.generators import Sine  # type: ignore[import]
+
     audio = Sine(440).to_audio_segment(duration=duration_ms - silence_ms)
     if silence_ms > 0:
         audio += AudioSegment.silent(duration=silence_ms)
     if clipping:
         # Artificially set max amplitude
         samples = audio.get_array_of_samples()
         max_val = max(samples)
-        audio = audio._spawn(bytes((max_val if i % 2 == 0 else 0 for i in range(len(samples)))))  # noqa: SLF001  # type: ignore[attr-defined]
+        audio = audio._spawn(
+            bytes((max_val if i % 2 == 0 else 0 for i in range(len(samples))))
```

Ruff (0.11.9)

Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)